### ISSUE TITLE
Monkey King will not spawn copies with Aganim's Scepter until you ult

### ISSUE THREAD

**Issue by realkarmakun** [ValveSoftware/Dota2-Gameplay#12089]
### Ability name

Wukong's Command

### Description

In the example match, brought Radiance and then Aganim's Scepter, copies did not spawn.
Was able to replicate it in the lobby. Copies didn't spawn until Monkey King ulted. They will spawn afterwards until the Monkey King's death. In the example match copies started to spawn at 29:15 (then stopped after death again)

### Example Match ID (and possibly Timestamp)

7345469179 - 19:15

**Comment by irismus** [comment]
#12064

**Comment by LPSwitch** [comment]
In my case, I've noticed that Aganims copies only spawn in either conditions:
1.) Wukong's Command is on cooldown.
2.) Wukong's Command is leveled up (works even when it is not on cooldown). Note that the 4 sec spawn interval still applies. so leveling Wukong's Command twice in quick succession will not spawn another Monkey. You must wait another 4 seconds before leveling up Wukong's Command to spawn another copy off cooldown.

**Comment by pablopeguero** [comment]
this just lost me my game. Im pretty sure it was like this yesterday and i didnt notice till my game today. This is game losing and i honestly cant belive it hasnt been fixed.

**Comment by realkarmakun** [comment]
> In my case, I've noticed that Aganims copies only spawn in either conditions: 1.) Wukong's Command is on cooldown. 2.) Wukong's Command is leveled up (works even when it is not on cooldown). Note that the 4 sec spawn interval still applies. so leveling Wukong's Command twice in quick succession will not spawn another Monkey. You must wait another 4 seconds before leveling up Wukong's Command to spawn another copy off cooldown.
> 
> TLDR; In order to have access to Aghanims monkeys, you would need to always have Wukong's Command on cooldown. If it goes off cooldown, you have to cast it again for Aghanims monkeys to spawn again.
> 
> Also note that dying has no relation to this bug. Dropping or backpacking then picking up the Aghanims doesn't work either. Wukong's Command just needs to be on cooldown.

Didn't have that. Copies was not spawning even when leveled up. Also after ult and it's cooldown copies did spawn until the death as described in my case.

**Comment by WizeWanter** [comment]
this just lost me my game

**Comment by tinaungkhant** [comment]
#12150
